### ISSUE TITLE
Blinken LEDs - WLAN and Clock-Symbol

### ISSUE THREAD

**Issue by stoffelll** [toremick/shorai-esp32#10]
From time to time I see two blinking LEDs on the indoor unit - see video.

After unpowering the nodeMCU and reconnecting it, the blinking stopps after some seconds.
Is there any way to debug the unit to identify the root cause?

Thanks and best regards

https://user-images.githubusercontent.com/5340003/138935008-a09d2fc4-f297-410f-936c-922ac53b3b3f.mp4

SL

**Comment by MrTheBarbarian** [comment]
I had the same problem when I tried to turn on a debug function in main/mqtt_as.py. On line 95 have to be  `DEBUG = False` Whe I changed on True, I got the same issue.

**Comment by stoffelll** [comment]
thanks, I'll check it :)

**Comment by stoffelll** [comment]
Meanwhile I double checked the source and DEBUG was false already.

Never the less I recognized that this blinking was occuring, when I had a network issue (hotspot didn't had connectivity to the rest of my network -> so no dhcp services or connection to mqtt). So I restarted my AccessPoint and the blinking stopped automatically after 1-2 minutes without doing anything. In my fork I added some small error handling and automatic restart of the ESP8266 - so I guess is solving the underlying issue.

**Comment by christianwicke** [comment]
I had the same problem: The two LEDs were blinking.
I was able to track it down: It is some a bug that prevents my esp32 to reconnect. I don't know whether it is a software bug or a hardware bug in my esp32.
I found out that sleeping for one second between reconnect-attempts works as a workaround. I provided this [patch to mqtt_as](https://github.com/peterhinch/micropython-mqtt/pull/154) which Peter Hinch accepted in the master branch of mqtt_as.

I'm using this on my [fork](https://github.com/christianwicke/shorai-esp32/). (To use the bigger mqtt_as 0.8.2, I had to migrate ota_updater to mrequests. The old HttpClient would run out of memory during download.) 
I didn't see any blinking LEDs for a almost week now, while before I saw them more or less daily.
